### PR TITLE
Add support for whois.nic.club

### PR DESCRIPTION
**Pull request by yastupin**
Not sure this one follows the best practices.
Feel free to comment on what's not optimal and I'll update the PR :)

### CODE REVIEW

File: spec/fixtures/responses/whois.nic.club/club/status_registered.txt [hunk 1]
```diff
@@ -0,0 +1,96 @@
+[Querying whois.nic.club]
```
Comment (weppos) [inline spec/fixtures/responses/whois.nic.club/club/status_registered.txt]: Please use the tool `./utils/mkwhois.rb` to generate the fixtures. These debug notes should not be part of the output.

File: lib/whois/parsers/whois.nic.club.rb [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+#
+# Copyright (c) 2009-2015 Simone Carletti <weppos@weppos.net>
+#++
+
+require_relative 'base'
+
+module Whois
+  class Parsers
+    # Parser for whois.nic.club server.
+    class WhoisNicClub < Base
```
Comment (weppos) [inline lib/whois/parsers/whois.nic.club.rb]: The whois.nic.club is probably a `BaseIcannCompliant` parser.

Comment (yastupin) [reply lib/whois/parsers/whois.nic.club.rb]: Right, looks like it is almost compliant now. It wasn't the case at all when I first wrote this parser ;)

Comment (weppos) [reply lib/whois/parsers/whois.nic.club.rb]: Thanks!

I'll merge it in the next few days.

**Comment by tiagoloureiro89** [comment]
Simone, only now saw this thread. Have developed a solution based on .biz solution ( same organization as .club ). Kindly see if it is of any use: https://github.com/weppos/whois-parser/pull/114

**Pull request closed without merging**